environs/config/config.go

```diff
@@ -1706,4 +1723,9 @@ data of the store. (default false)`,
 		Type:        environschema.Tstring,
 		Group:       environschema.EnvironGroup,
 	},
+	BackupDirKey: {
+		Description: "Directory used to store temporal files when creating a backup",
```

Comment:
... to store the backup working directory.

Reply:
Fixed